src/core/time.d

```diff
@@ -1667,6 +1667,52 @@ public:
         }
     }
 
+    /++
+        Converts `Duration` to a `timeval` structure.
```

Comment:
Please use Ddoc style comments.

Reply:
I'm not sure what do you mean by Ddoc style comments.
For me any /++, /** or /// comment is Ddoc comment.

Reply:
The only improvement that I could see is that you could add a [`Returns: `](https://dlang.org/spec/ddoc.html#standard_srction) section.

Reply:
Walter was asking for DDOC sections, like @ZombineDev says:

```D
/++
       Returns: A `timeval` structure representing this `Duration`.
++/
```
And do the same to the other function as well.

Reply:
@schveiguy I just did it the same way as in toString()
```
Converts this `Duration` to a `string`
```
And doc comments for toString() does not have ddoc sections.

Reply:
I honestly see zero point in using `Returns` when the whole comment is telling you what the thing is returning. Sections make sense when you're breaking up the comment but not so much when the comment is simple.